source/app.d

```diff
@@ -104,39 +105,39 @@ void main(string[] _args)
 
 	envVars["FD_CMDLINE"] = cmdline;
 
-    File outStream = stdout;
-    File errStream = stderr;
+	File outStream = stdout;
+	File errStream = stderr;
 
-    if(options.useStdOutFile)
-        outStream = File(options.stdoutFile, "a+");
+	if(options.useStdOutFile)
+		outStream = File(options.stdoutFile, "a+");
 
-    if(options.useStdErrFile)
-        errStream = File(options.stderrFile, "a+");
+	if(options.useStdErrFile)
+		errStream = File(options.stderrFile, "a+");
 
 	while((options.max == -1 || (options.max-- > 0)) && canRestart)
 	{
 		log("Starting: '%s'", cmdline);
 		uptime.start();
 
-        auto pid = spawnProcess(cmdline, std.stdio.stdin, outStream, errStream,
-            null, Config.retainStdout);
-            
-        auto exitCode = wait(pid);
+		auto pid = spawnProcess(_args[1 .. $], std.stdio.stdin, outStream, errStream,
```

Comment:
`spawnProcess` either takes a single string which is the name of a process, or an array of strings that is the process (optionally) followed by args.